### DISCUSSION TITLE
General Questions

### DISCUSSION THREAD

**Discussion by plk0507** [mifi/lossless-cut#1299] [General]
Hi,
Thank you so much for building this. I have a couple of questions.
1) How do you change the output file name? I see it's possible, but I cannot figure it out.
2) I have a bunch of errors that say, "DEVTOOLS failed to parse SourceMap." Is this a big deal?
3) When I play a movie that I've cut, the time in the players shows it beginning at the cut time. Is that ok? I would expect the beginning to be zero.
Thanks,

**Comment by plk0507** [comment]
I figured out the output file name. Thank you.

**Reply by mifi** [reply]
Hi

2. Nope
3. Maybe you need to experiment with keyframe/normal cut as well as export options like avoid_negative_ts to see if it makes a difference
